### PR TITLE
Feature/refactor

### PR DESCRIPTION
**Pull request by John2496**

### CODE REVIEW

**Review by PHimmel — CHANGES_REQUESTED** [review]
There are unused dependencies, please fix and resubmit. What am I paying you for anyway?

File: scraper/soup.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from bs4 import BeautifulSoup as soup
+import urllib
```
Comment (PHimmel) [inline scraper/soup.py:2]: Do we need this?

File: scraper/http.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from bs4 import BeautifulSoup as soup
+import urllib
+import urllib.request
+import urllib.error
+import re
+from textblob import TextBlob as blob
+import time
```
Comment (PHimmel) [inline scraper/http.py:7]: Do we need this?

File: scraper/http.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+from bs4 import BeautifulSoup as soup
+import urllib
+import urllib.request
+import urllib.error
+import re
```
Comment (PHimmel) [inline scraper/http.py:5]: Needed?

Comment (John2496) [reply scraper/http.py:5]: Resolved.

File: scraper/http.py (hunk 2 above)
Comment (John2496) [reply scraper/http.py:7]: Resolved.

File: scraper/soup.py (hunk 1 above)
Comment (John2496) [reply scraper/soup.py:2]: Resolved.

**Pull request merged**